src/main/java/racingcar/util/RandomGenerator.java

```diff
@@ -0,0 +1,17 @@
+package racingcar.util;
+
+import static racingcar.constants.Symbol.ZERO;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+public class RandomGenerator {
+    private static final Integer MOVABLE_MIN_NUMBER = 4;
+    private static final Integer RANDOM_MAX_NUMBER = 9;
+    public static boolean Movable() {
+        return generateRandomNumber() >= MOVABLE_MIN_NUMBER;
+    }
+    private static Integer generateRandomNumber(){
+        return Randoms.pickNumberInRange(ZERO, RANDOM_MAX_NUMBER);
+    }

```

Comment:
p4 

다른 곳은 공백이 잘되어있는데 여기는 바쁘셔서 놓치신 것 같습니당😆

```suggestion
    private static final Integer MOVABLE_MIN_NUMBER = 4;
    private static final Integer RANDOM_MAX_NUMBER = 9;
    
    public static boolean Movable() {
        return generateRandomNumber() >= MOVABLE_MIN_NUMBER;
    }
    
    private static Integer generateRandomNumber(){
        return Randoms.pickNumberInRange(ZERO, RANDOM_MAX_NUMBER);
    }

```

Reply:
헉.. 이런 부분도 봐주시다니 완전 매의 눈이십니다..! 👍 

해당 코드에서 코드 정렬 기능을 사용하지 못한 것 같습니다..!
앞으로 더 신경쓰도록 하겠습니다!